package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@uportal/app-framework",
-  "version": "12.2.0",
+  "version": "12.3.0",
```

Eh. I'm not sure about this bumping the `version` in the PR that adds a feature rather than in the process of cutting a release. This is going to make [the instructions for cutting the release](https://github.com/uPortal-Project/uportal-app-framework/blob/master/docs/releasing.md#node-release) not quite right, since those instructions involve running `npm version` to bump the version number, isn't it?

```suggestion
  "version": "12.2.0",
```

---

Thanks, I had not read those instructions.  I changed them back to 12.2.0 for the JS artifact and 12.2.1-SNAPSHOT for the Java artifact.